x-pack/plugins/observability/public/components/alerts_flyout/use_get_alert_flyout_components.tsx

```diff
@@ -19,6 +22,7 @@ export { AlertsFlyout } from './alerts_flyout';
 export const useGetAlertFlyoutComponents = (
   observabilityRuleTypeRegistry: ObservabilityRuleTypeRegistry
 ) => {
+  const isSLODetailsPage = useRouteMatch(SLO_DETAIL_PATH);
```

Comment:
This component `alerts_flyout` is not specific to SLO, yet we add a very specific SLO condition here. It does not seem to be the right place. 
But in the other hand, I don't have a better approach because of the way AlertTable / AlertTableRegistry and all this works together...